amazoncorretto-11-debian/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM debian:stable-slim
+ARG EXTRA_TAG_SUFFIXES=bullseye
```

Comment:
I'd suggest either just scraping the `FROM debian` line for the release name or putting it in a comment instead so that it can't affect the build.

Reply:
That is fair, I can make the `EXTRA_TAG_SUFFIXES` be in a comment instead.

I would like to avoid parsing the `FROM` for figuring out the release since it will possibly become a bit more specific for debian at that point IMO whereas the other way creates a generic way to specify extra suffixes for an image.

Reply:
Just pushed up a commit changing it to be a comment instead of an ARG.

cc @carlossg